src/app/util/ember-compatibility-functions.cpp

```diff
@@ -366,10 +367,9 @@ CHIP_ERROR ReadSingleClusterData(FabricIndex aAccessingFabricIndex, const Concre
         return SendFailureStatus(aPath, attributeReport, Protocols::InteractionModel::Status::UnsupportedAttribute, nullptr);
     }
 
-    AttributeAccessInterface * attrOverride = findAttributeAccessOverride(aPath.mEndpointId, aPath.mClusterId);
```

Comment:
Please add a TODO comment here about needing to do ACL checks for the override case.  I think the ACL check should just be after the `if (attributeMetadata == nullptr)` block, but don't want to hold this up for that at this point....

Reply:
I'll move it back up. I just wasn't sure about exactly how much metadata checking is going on before the ember functions are actually called, though I agree they SHOULD ultimately do all the checks (exists? readable?) that we need. However I'm also leery of the attribute overrides and chunking. It doesn't hurt to move the check back up for now, and if all the metadata checks are good (more like the write code), then we're done.

Reply:
Done.